wetterdienst/radolan.py

```diff
@@ -0,0 +1,67 @@
+""" Functions for DWD RADOLAN collection """
+from pathlib import Path
+from typing import Union
+import datetime as dt
+
+import pandas as pd
+
+from wetterdienst.constants.access_credentials import DWDCDCDataPath
+from wetterdienst.constants.metadata import DWD_FOLDER_MAIN
+from wetterdienst.data_storing import store_radolan_data, restore_radolan_data
+from wetterdienst.download.download import download_radolan_data
+from wetterdienst.enumerations.column_names_enumeration import DWDMetaColumns
+from wetterdienst.enumerations.parameter_enumeration import Parameter
+from wetterdienst.enumerations.time_resolution_enumeration import TimeResolution
+from wetterdienst.enumerations.period_type_enumeration import PeriodType
+from wetterdienst.exceptions.start_date_end_date_exception import DatetimeOutOfRangeError
+from wetterdienst.indexing.file_index_creation import create_file_index_for_radolan
+
+
+def collect_radolan_data(
+        datetime: Union[str, dt.datetime],
+        time_resolution: TimeResolution,
+        folder: Union[str, Path] = DWD_FOLDER_MAIN,
+):
+    datetime = pd.to_datetime(datetime).replace(tzinfo=None)
```

Comment:
I think passing '1/1/2020 - 10:15:00' as string should not be possible because there are a lot of mistakes you can do by passing datetim strings. 
One possible solution could be to allow string passing only for ISO-Format. This can be easiely checked.

Reply:
Good point! I'll check that one out!

Reply:
I couldnt really find a convincing method. pd.to_datetime allows to give it anything even a datetime. Using more accurate methods one needs to check first if the argument isn't already a datetime. Thus I switched back to pd.to_datetime. The user should be aware of entering meaningful datetimes. Unfortunately pd.to_datetime also doesnt offer limiting recognized datetime strings.